zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/ObjectenApiPlugin.kt

```diff
@@ -59,6 +59,23 @@ class ObjectenApiPlugin(
         )
     }
 
+    fun getObjectsByObjectTypeIdWithSearchParams(
+        objecttypesApiUrl: URI,
+        objectsApiUrl: URI,
```

Comment:
The `objectsApiUrl` doesn't need to be an function-argument. You can take it from `@PluginProperty` defined above